Source/iop/Iop_McServ.cpp

```diff
@@ -91,7 +92,6 @@ void CMcServ::CountTicks(uint32 ticks, CSifMan* sifMan)
 	moduleData->pendingCommandDelay -= std::min<uint32>(moduleData->pendingCommandDelay, ticks);
 	if(moduleData->pendingCommandDelay == 0)
 	{
-		assert(moduleData->pendingCommand == CMD_ID_GETINFO);

```

Comment:
We should check that we have a pending command here (ie.: `assert(moduleData->pendingCommand != CMD_ID_NONE`).

Reply:
This could never occur, as there is a `if(moduleData->pendingCommand == CMD_ID_NONE) return;` check above.